source/vocab/concepts.ttl

```diff
@@ -117,7 +117,7 @@
     owl:equivalentProperty skos:definition ;
     owl:equivalentProperty madsrdf:definitionNote .
 
-:exactMatch a owl:ObjectProperty ;
+:exactMatch a owl:SymmetricProperty ;
```

Comment:
```suggestion
:exactMatch a owl:ObjectProperty, owl:SymmetricProperty ;
```

I think we have code checking if properties are (`owl:`)`ObjectProperty` both in the whelk and lxlviewer...